src/main/resources/output/debugResultsMatchingRule.csv

```diff
@@ -0,0 +1 @@
+"MatchingRule","Record1Identifier","Record2Identifier","TotalSimilarity","IsMatch","0-CarManufacturerComparator_LowerCase-record1Value","0-CarManufacturerComparator_LowerCase-record2Value","0-CarManufacturerComparator_LowerCase-record1PreprocessedValue","0-CarManufacturerComparator_LowerCase-record2PreprocessedValue","0-CarManufacturerComparator_LowerCase-similarity","0-CarManufacturerComparator_LowerCase-postproccesedSimilarity"
```

Comment:
Add those files to gitignore and remove them from git. We do not need the debug output in here.

Reply:
already in there?....